StyleGuide.md

```diff
@@ -535,52 +540,62 @@ Do not use underlined text in CockroachDB docs. If it seems beneficial to emphas
 
 Whenever a CockroachDB feature is referenced, provide a link to the relevant documentation. You can also provide links to external resources, but only if the resource is confirmed to be accurate by a technical reviewer or the author is a Cockroach Labs SME and no CockroachDB documentation covers the topic.
 
+Links are marked with inline text surrounded by square brackets followed by the link address in parentheses.
+
 Avoid using non-descriptive link names such as `here`, `this page`, or `go`.
 
-Use Markdown reference-style links when several parts of the same page refer to the same target URL (e.g., [Release Notes](releases/v22.1.html)).
+Use Markdown reference-style links when several parts of the same page refer to the same target URL.
 
-For websites that automatically localize pages, avoid using localization elements directly within the URL. For example:
-
-- GitHub
-  - Instead of `https://docs.github.com/**en/**graphql/overview/explorer`
-  - Use `https://docs.github.com/graphql/overview/explorer`
-- Wikipedia
-  - Instead of `https://en.wikipedia.org/wiki/SQL:2011`
-  - Use `https://www.wikipedia.org/wiki/SQL:2011` or `https://wikipedia.org/wiki/SQL:2011`
+#### Link capitalization
 
 Link capitalization should match our [capitalization rules](#capitalization-rules) for page titles and headers:
 
 - **Use title case** when referring to the linked doc by its page title (e.g., "See __Best Practices__ for more information").
 - **Use sentence case** when referring to the linked doc by one of its headers (e.g., "See __Clock synchronization__ for further guidance").
 - **Use sentence case** - when referring to a linked doc without explicitly citing a page title or header (e.g., "[…] follow the __identifier rules__ when creating […]").
 
-Links are marked with inline text surrounded by square brackets followed by the link address in parentheses. If you are including a relative (i.e., internal) link:
+#### Links to CockroachDB docs pages in the same folder
 
-- To link to another page in the docs, use just the name of the file.
+To link to a page within the same folder (e.g., a page in `v23.1` to another page in `v23.1`), use the [Jekyll link syntax](https://jekyllrb.com/docs/liquid/tags/#links).
```

Comment:
Do you think we need to clarify that this doesn't apply to files within a subfolder of the same folder? So, you need to use the URL if it's a `v23.1/abc.md` file linking to a `v23.1/architecture/xyz.md` file?

Reply:
I think that's a great idea. I'll add that!